Docs: What’s new & Upgrade guide v11.1 (#89075)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v11-1.md

---
description: Feature and improvement highlights for Grafana v11.1
keywords:
  - grafana
  - new
  - documentation
  - '11.1'
  - release notes
labels:
products:
  - cloud
  - enterprise
  - oss
title: What's new in Grafana v11.1
weight: -43
---

<!-- vale GoogleWe = NO -->
<!-- vale We = NO -->

# What’s new in Grafana v11.1

> Welcome to Grafana 11.1! This release contains some major improvements: most notably...
> 
> <!--{{< youtube id=" " >}}-->
> 
> For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md). For the specific steps we recommend when you upgrade to v11.1, check out our [Upgrade Guide](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/upgrade-guide/upgrade-v11.1/).
> 
> <!--## Breaking changes

Author: ```suggestion
Welcome to Grafana 11.1! This release contains some major improvements. Most notably, we've adding cell text wrapping to table visualizations, a much-requested feature. We've made redesigned the Alerting settings page and made it easier to set rule-specific silences. A number of accessibility improvements are included in this release, making Grafana easier to use for everyone. And lastly, XY charts are now generally available.

<!-- {{< youtube id=" " >}} -->

For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md). For the specific steps we recommend when you upgrade to v11.1, check out our [Upgrade Guide](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/upgrade-guide/upgrade-v11.1/).

<!-- ## Breaking changes
```

![An xy chart showing height weight distribution](/media/docs/grafana/panels-visualizations/screenshot-xy-charts-v11.0.png)

Over the past several months we’ve introduced multiple enhancements to the visualizations like auto mode, which now handles most scenarios that previously required manual configuration. Additionally, we've added better control over point styling and further improved performance. We’re excited to include XY chart as a first class citizen in the core Grafana visualization library. To learn more about the panel, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/xy-chart/).

To use the new XY chart visualization, you must first enable the `autoMigrateXYChartPanel` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/).

## Alerting

### Re-designed settings page for Alerting

> <!-- Gilles De Mey -->

Author: ```suggestion
<!-- Gilles DeMey -->
```

The new settings page provides you with a holistic view of where Grafana-managed alert instances are forwarded.

- Manage which Alertmanagers receive alert instances from Grafana-managed rules without navigating and editing data sources.
- Manage version snapshots for the built-in Alertmanager, which allows administrators to roll back unintentional changes or mistakes in the Alertmanager configuration.
- There is also a visual diff that compares the historical snapshot with the latest configuration to see which changes were made.

{{< figure src="/media/docs/alerting/alert-settings.png" alt="Alert settings page" >}}

### Alerting template selector

<!-- Sonia Aguilar -->

_Generally available in all editions of Grafana_

Enables you to easily choose which templates you want to use in your alert notification messages by adding a template selector in the Contact Points form.

Select from existing templates or enter a custom one for your specific needs. You can switch between the two tabs to access the list of available templates and copy them across to the customized version.

### Add OAuth2 to HTTP settings for vanilla Alertmanager / Mimir

> <!-- Gilles de Mey -->

Author: ```suggestion
<!-- Gilles deMey -->
```

XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables. XY charts are typically used to create scatter plots. You can also use them to create bubble charts where field values determine the size of each bubble:

![An xy chart showing height weight distribution](/media/docs/grafana/panels-visualizations/screenshot-xy-charts-v11.0.png)

Over the past several months we’ve introduced multiple enhancements to the visualizations like auto mode, which now handles most scenarios that previously required manual configuration. Additionally, we've added better control over point styling and further improved performance. We’re excited to include XY chart as a first class citizen in the core Grafana visualization library. To learn more about the panel, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/xy-chart/).

To use the new XY chart visualization, you must first enable the `autoMigrateXYChartPanel` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/).

## Alerting

> 
> <!-- TO DO - Confirm with Alerting folks that everything available in OSS is also available in Enterprise except for Rule specific silences with permissions and Rule specific silences with RBAC-->

Author: ```suggestion
```

### Re-designed settings page for Alerting

<!-- Gilles De Mey -->

_Generally available in all editions of Grafana_

The new settings page provides you with a holistic view of where Grafana-managed alert instances are forwarded.

- Manage which Alertmanagers receive alert instances from Grafana-managed rules without navigating and editing data sources.
- Manage version snapshots for the built-in Alertmanager, which allows administrators to roll back unintentional changes or mistakes in the Alertmanager configuration.
- There is also a visual diff that compares the historical snapshot with the latest configuration to see which changes were made.

{{< figure src="/media/docs/alerting/alert-settings.png" alt="Alert settings page" >}}

### Alerting template selector

<!-- Sonia Aguilar -->

> _Generally available in all editions of Grafana_

Author: @soniaAguilarPeiron - Can you please confirm that this is available in both OSS **and** Enterprise? It was only listed as being available in Cloud and OSS on the original Cloud entry.

Reviewer: Available in both OSS and Enterprise

Author: Thanks very much for checking on these!

Enables you to easily choose which templates you want to use in your alert notification messages by adding a template selector in the Contact Points form.

Select from existing templates or enter a custom one for your specific needs. You can switch between the two tabs to access the list of available templates and copy them across to the customized version.

### Add OAuth2 to HTTP settings for vanilla Alertmanager / Mimir

<!-- Gilles de Mey -->

> _Generally available in all editions of Grafana_

Author: @gillesdemey -  - Can you please confirm that this is available in both OSS **and** Enterprise? It was only listed as being available in Cloud and OSS on the original Cloud entry.

Reviewer: Available in both OSS and Enterprise

Configure OAuth2 authentication for any Alertmanager or Mimir receiver (called Contact Points in Grafana) through the user interface.

OAuth2 is not implemented for the Grafana built-in Alertmanager.

### Improved paused alert visibility

<!-- Tom Ratcliffe -->

> _Generally available in all editions of Grafana_

Author: @tomratcliffe - Can you please confirm that this is available in both OSS **and** Enterprise? It was only listed as being available in Cloud and OSS on the original Cloud entry.

Reviewer: Available in both OSS and Enterprise

Pause and resume alert rule evaluation directly from the Alert rules list and details view. This helps Improve visibility of when alert rules have been paused by displaying “Paused” as the alert rule state.

### Removes requirement of datasources:query permission for reading rules

<!-- William Wernert -->

> _Generally available in all editions of Grafana_

Author: @rwwiv - Can you please confirm that this is available in both OSS **and** Enterprise? It was only listed as being available in Cloud and OSS on the original Cloud entry.

Reviewer 2: This would actually be the other way around. The permission change technically applies to all versions, but is really only relevant for Cloud and Enterprise.

Author: ```suggestion
_Generally available in Grafana Enterprise and Cloud_
```

---
description: Feature and improvement highlights for Grafana v11.1
keywords:
  - grafana
  - new
  - documentation
  - '11.1'
  - release notes
labels:
products:
  - cloud
  - enterprise
  - oss
title: What's new in Grafana v11.1
weight: -43
---

<!-- vale GoogleWe = NO -->
<!-- vale We = NO -->

# What’s new in Grafana v11.1

Welcome to Grafana 11.1! This release contains some minor improvements following on the big announcements in Grafana 11.0. We've adding cell text wrapping to table visualizations, a much-requested feature. We've also redesigned the Alerting settings page and made it easier to set rule-specific silences. A number of accessibility improvements are included in this release, making Grafana easier to use for everyone. And lastly, the XY chart is now generally available.

> <!-- {{< youtube id=" " >}} -->

Author: ```suggestion
<!-- {{< youtube id="gjQX9-7Hsfs" >}} -->
```

> <!-- {{< youtube id="gjQX9-7Hsfs" >}} -->

Author: ```suggestion
{{< youtube id="gjQX9-7Hsfs" >}}
```

<!-- Gilles -->

_Generally available in all editions of Grafana_

The new settings page provides you with a holistic view of where Grafana-managed alert instances are forwarded.

- Manage which Alertmanagers receive alert instances from Grafana-managed rules without navigating and editing data sources.
- Manage version snapshots for the built-in Alertmanager, which allows administrators to roll back unintentional changes or mistakes in the Alertmanager configuration.
- There is also a visual diff that compares the historical snapshot with the latest configuration to see which changes were made.

{{< figure src="/media/docs/alerting/alert-settings.png" alt="Alert settings page" >}}

### Alerting template selector

<!-- Sonia Aguilar -->

_Generally available in all editions of Grafana_

Enables you to easily choose which templates you want to use in your alert notification messages by adding a template selector in the Contact Points form.

Select from existing templates or enter a custom one for your specific needs. You can switch between the two tabs to access the list of available templates and copy them across to the customized version.

> 

Author: ```suggestion

Learn more in [Use notification templates](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/use-notification-templates/)

```

_Generally available in all editions of Grafana_

More easily create silences directly from the Alert rule list view or detail page.

These rule-specific silences are guaranteed to only apply to a single rule and permissions to read, create, update or delete are tied to a user’s permissions for that rule.

### Rule-specific silences with RBAC

<!-- Tom Ratcliffe -->

_Generally available in Grafana Enterprise and Cloud_

Manage silences through Role-Based Access Control (RBAC). In addition to the Grafana open source functionality in **Rule-specific silences with permissions**, you can choose who can create, edit, and read silences using the following permissions:

- Users with the **alert.silences:create permission**, scoped within a folder, are able to create silences for rules contained within that folder and its subfolders
- Users with the **alert.silences:read permission**, scoped within a folder, are able to read silences for rules contained within that folder and its subfolders, and general silences
- Users with the **alert.silences:write permission**, scoped within a folder, are able to expire and recreate silences for rules contained within that folder and its subfolders

> 

Author: ```suggestion

Learn more about [managing access using roles](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/set-up/configure-rbac/access-roles/)

```

To use the new XY chart visualization, you must first enable the `autoMigrateXYChartPanel` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/).

## Alerting

### Re-designed settings page for Alerting

<!-- Gilles -->

_Generally available in all editions of Grafana_

The new settings page provides you with a holistic view of where Grafana-managed alert instances are forwarded.

- Manage which Alertmanagers receive alert instances from Grafana-managed rules without navigating and editing data sources.
- Manage version snapshots for the built-in Alertmanager, which allows administrators to roll back unintentional changes or mistakes in the Alertmanager configuration.
- There is also a visual diff that compares the historical snapshot with the latest configuration to see which changes were made.

{{< figure src="/media/docs/alerting/alert-settings.png" alt="Alert settings page" >}}

> 

Author: ```suggestion

Learn more in the [Alertmanager configuration documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/).
```

### Alerting template selector

<!-- Sonia Aguilar -->

_Generally available in all editions of Grafana_

Enables you to easily choose which templates you want to use in your alert notification messages by adding a template selector in the Contact Points form.

Select from existing templates or enter a custom one for your specific needs. You can switch between the two tabs to access the list of available templates and copy them across to the customized version.

> Learn more in [Use notification templates](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/use-notification-templates/)

Author: ```suggestion
Learn more in [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/manage-contact-points/)
```

### Add OAuth2 to HTTP settings for vanilla Alertmanager / Mimir

<!-- Gilles -->

_Generally available in all editions of Grafana_

> Configure OAuth2 authentication for any Alertmanager or Mimir receiver (called Contact Points in Grafana) through the user interface.

Reviewer 3: ```suggestion
Configure OAuth2 authentication for any Alertmanager or Mimir receiver (called Contact Points in Grafana) through the user interface. Learn more about alerting contact points in [supported Contact Point integrations](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/#list-of-supported-integrations) and find the reference for Oauth2 fields in the [Prometheus Alertmanager docs for Oauth2](https://prometheus.io/docs/alerting/latest/configuration/#oauth2).
```

> OAuth2 is not implemented for the Grafana built-in Alertmanager.

Reviewer 3: ```suggestion
Note that OAuth2 is not yet implemented for the Grafana built-in Alertmanager.
```

### Improved paused alert visibility

<!-- Tom Ratcliffe -->

_Generally available in all editions of Grafana_

> Pause and resume alert rule evaluation directly from the Alert rules list and details view. This helps Improve visibility of when alert rules have been paused by displaying “Paused” as the alert rule state.

Author: ```suggestion
Pause and resume alert rule evaluation directly from the Alert rules list and details view. This helps Improve visibility of when alert rules have been paused by displaying “Paused” as the alert rule state.

Learn more in the [alert rules documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/manage-notifications/view-alert-rules/)
```

### Removes requirement of datasources:query permission for reading rules

<!-- William Wernert -->

_Generally available in Grafana Enterprise and Cloud_

Fetching a rule group no longer requires the **datasources:query** permission for every data source used by the rules within that group. Now, the only requirements are **alert.rules:read** and **folders:read** for the folder the group is contained in.

Note: **datasources:query** is still required to preview an alert rule, regardless of alert rules and folders permissions.

> 

Author: ```suggestion

Learn more in the [RBAC configuration documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/set-up/configure-rbac/)
```

_Generally available in all editions of Grafana_

More easily create silences directly from the Alert rule list view or detail page.

These rule-specific silences are guaranteed to only apply to a single rule and permissions to read, create, update or delete are tied to a user’s permissions for that rule.

### Rule-specific silences with RBAC

<!-- Tom Ratcliffe -->

_Generally available in Grafana Enterprise and Cloud_

Manage silences through Role-Based Access Control (RBAC). In addition to the Grafana open source functionality in **Rule-specific silences with permissions**, you can choose who can create, edit, and read silences using the following permissions:

- Users with the **alert.silences:create permission**, scoped within a folder, are able to create silences for rules contained within that folder and its subfolders
- Users with the **alert.silences:read permission**, scoped within a folder, are able to read silences for rules contained within that folder and its subfolders, and general silences
- Users with the **alert.silences:write permission**, scoped within a folder, are able to expire and recreate silences for rules contained within that folder and its subfolders

> Learn more about [managing access using roles](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/set-up/configure-rbac/access-roles/)

Author: ```suggestion
Learn more about [managing access using roles](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/set-up/configure-rbac/access-roles/).
```

{{< youtube id="gjQX9-7Hsfs" >}}

For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md). For the specific steps we recommend when you upgrade to v11.1, check out our [Upgrade Guide](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/upgrade-guide/upgrade-v11.1/).

<!-- ## Breaking changes

For Grafana v11.0, we've also provided a list of [breaking changes](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/breaking-changes/breaking-changes-v11-0) to help you upgrade with greater confidence. For information about these along with guidance on how to proceed, refer to [Breaking changes in Grafana v11.0](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/breaking-changes/breaking-changes-v11-0/).-->

## Dashboards and visualizations

### Visualization ease of use improvements

<!-- Drew Slobodnjak, Kyle Cunningham -->

> _Generally available in all editions of Grafana_
> 

Author: ```suggestion
```

We’ve made a number of small improvements to the data visualization experience in Grafana.

#### Table cell text wrapping

> 

Author: ```suggestion

_Available in public preview in all editions of Grafana_

```
